thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomalydetection/function/BackwardAnomalyFunctionUtils.java

```diff
@@ -47,7 +47,8 @@
     }
     // Sort time series by their start time in reversed natural order, i.e., the latest time series
     // is arranged in the front of the list
-    Collections.sort(timeSeriesList, new TimeSeriesStartTimeComparator().reversed());
+    Collections.sort(timeSeriesList, new TimeSeriesStartTimeComparator());
+    Collections.reverse(timeSeriesList);
```

Comment:
Can be combined as Collections.sort(timeSeriesList, Collections.reverse(new TimeSeriesStartTimeComparator()));